StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1625ElementDocumentationMustNotBeCopiedAndPasted.cs

```diff
@@ -50,6 +50,7 @@
     /// </code>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:question: Would it be reasonable to provide a code fix that allows you to remove the duplicated text? It's certainly one where we wouldn't provide a `FixAllProvider`, but for removing invalid text prior to adding the new text it might be handy.